tests/test_mail.py

```diff
@@ -198,3 +220,11 @@ def test_build_sg_email_w_custom_args(self):
                  'personalizations': [{'subject': ''}],
                  'subject': ''}
             )
+            
+    def test_send_messages_error(self):
+        mock_client = MockClient(self.host)
```

Comment:
`self.host` doesn't appear to be assigned anywhere, so this test always throws an error.